src/components/Comment/CommentActions.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {css} from 'glamor'
+import {MdShare, MdKeyboardArrowDown, MdKeyboardArrowUp} from 'react-icons/lib/md'
+import {Label} from '../Typography'
+import MdMoreHoriz from '../Icons/MdMoreHoriz'
```

Comment:
FYI, this guy actually works for me:
https://github.com/gorangajic/react-icons/blob/master/md/more-horiz.js

Reply:
Ok, I was using http://gorangajic.github.io/react-icons/md.html to see which icons are available.